hack/deploy.sh

```diff
@@ -60,20 +60,12 @@ fi
 if [ "$INSTALL_METERING" == "true" ]; then
     echo "Installing metering"
     install_metering "${INSTALL_METHOD}"
-
-    echo
-    echo "Waiting for metering-operator pod to start termination"
-    # we just check until there's a non-ready container then the loop below this will check for readiness
-    until [ "$(kubectl -n $METERING_NAMESPACE get pods -l app=metering-operator -o json | "$FAQ_BIN" -f json -o json -M -c -r '.items | map(.status.containerStatuses[].ready) | all')" == "false" ]; do
```

Comment:
This is racy because the operator could become ready before this sees it's unready, so I removed it. It's mostly for continuous upgrade jobs which we'll be removing soon when we swap CI systems anyways. Can revisit later.